package.json

```diff
@@ -38,6 +38,7 @@
     "gl-matrix": "2.3.2",
     "handlebars": "^4.0.5",
     "jszip": "2.6.0",
+    "lodash.get": "^4.3.0",
```

Comment:
I would prefer not adding a new dep and just using the previous approach for now. We can tackle this cleanup after